### ISSUE TITLE
Duplicate Name Warning on Domino Visual Editor File Prompts

### ISSUE THREAD

**Issue by JoelProminic** [Moonshine-IDE/Moonshine-IDE#1220]
While testing #1219 today, I found that there was no warning in the New Shared Column prompt when the prepopulated name was a duplicate. Instead, I only see that the Create button is disabled:
![image](https://github.com/Moonshine-IDE/Moonshine-IDE/assets/24532731/1cbe7014-3ecf-4f64-b698-951bf44ade10)

I thought this was implemented for Forms and Views already, but when I tested this I found the same behavior.  I eventually found the expected behavior on New File:

![image](https://github.com/Moonshine-IDE/Moonshine-IDE/assets/24532731/67032f80-e8b1-4162-bbed-da5b2673d1a6)

(The message in this case should be "ActionTracker.form already exists" - remove the "is")

For the Domino objects, we can use a similar format:

> %file% already exists.

However, we'll need to be careful with the escaped names (i.e. `\` or `/` encoded as `_5c`).

**Comment by JoelProminic** [comment]
Type | Behavior
-- | --
Form | Works
View | Works
View with `\` | BUG:  duplicate not detected
Subform | BUG:  No message
Page | BUG:  No message
Shared Field (Was this supposed to be hidden?) | BUG:  No message
Shared Action (Was this supposed to be hidden?) | BUG:  No message
Shared Column (project tree) | BUG:  No message
Shared Column with `\` (project tree) | BUG:  duplicate not detected
Shared Column (from view) | Works


The current message is

> TestID.column is already exists.

Instead this should be:

> TestID.column already exists.


I also noticed that the prompts were updated with default names for the Forms and Views.   I'm not sure I like this change, but I find it easy to clear the names for now.

**Comment by JoelProminic** [comment]
@feather812002 pointed out that I did not have `\` tests for Forms and Subforms.   To my knowledged we have not added support for this yet.  I don't think we have examples for these cases in the Prominic databases, but they may be used in names.nsf.  For now, we should restrict `\` and `/` in the prompts for Forms and Subforms.  @feather812002 already made an update for this.

While going through the test matrix I found a similar problem for Pages.  The slashes should be restricted in this case as well.

Otherwise, the updates look good.  The message has also been fixed.

Type | Duplicate Name | Slashes
-- | -- | --
Form | Detected | Restricted
View | Detected | Detected
Subform | Detected | Restricted
Page | BUG:  Not Detected | BUG:  Should be restricted for now
Shared Field | Hidden | N/A
Shared Action  | Hidden | N/A
Shared Column (project tree) | Detected | Detected
Shared Column (from view) | Detected | Detected
